### PR TITLE
fix: cleanup Transport and add keepalive

### PR DESCRIPTION
**Pull request by zzmp**

### CODE REVIEW

**Comment by zzmp** [comment]
I believe it's fine to `useBatch` unless and until attribution is implemented (see https://github.com/Uniswap/interface/pull/5027 for a previous attempt).

**Comment by lynnshaoyu** [comment]
I think attribution is enabled in the SDK by default tho (it's supposed to be) but its not working for us for whatever reason. That PR is my attempt to get it working :S

File: src/analytics/index.ts [hunk 1]
```diff
@@ -35,6 +35,10 @@ export function initializeAnalytics(
         region: false,
         dma: false, // designated market area
       },
+      // Configure the SDK to use batched events (to reduce network contention)
+      // by grouping events from the same render cycle
+      flushIntervalMillis: 50, // assuming 20fps renders
+      useBatch: true,
```
[the remark this answers is not in the document]
Comment (lynnshaoyu) [reply src/analytics/index.ts:41]: yeah tbh im not comfortable adding useBatch given how many bugs we are trying to figure out related to referral params / attribution tracking not working in amplitude rn.

**Review by dannydaniil — APPROVED** [review]
lgtm

**Pull request merged**